packages/react-components/react-button/library/src/components/SplitButton/useSplitButtonStyles.styles.ts

```diff
@@ -170,6 +172,14 @@ const useRootStyles = makeStyles({
       },
     },
   },
+
+  largeChevron: {
+    [`& .${menuButtonIconClass}`]: {
```

Comment:
And then use it here as:

```suggestion
    [`& .${menuButtonClassNames.menuIcon}`]: {
```